dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/planner/QueryPlan.java

```diff
@@ -97,4 +97,37 @@ public void setUser( User user )
             nonPersistedQuery.setUser( user );
         }
     }
+
+    public static final class QueryPlanBuilder
+    {
+        private Query persistedQuery;
+
+        private Query nonPersistedQuery;
+
+        private QueryPlanBuilder()
+        {
+        }
+
+        public static QueryPlanBuilder aQueryPlan()
```

Comment:
Rename to `instance` ? Perhaps a bit more standard.

Reply:
Renamed to `newBuilder()`, same as `DataQueryParams`